cdap-ui/app/cdap/components/RulesEngineHome/RulesTab/Rule/Rule.scss

```diff
@@ -32,6 +32,15 @@ $drag_bars_color: gray;
       padding-left: 0;
     }
 
+    .rule-name-container {
+      display: flex;
```

Comment:
if there is only one child for this container, do you need to set it to display flex?

Reply:
Its not single element. It has svg arrow and the name.